app/lib/filterServices.js

```diff
@@ -4,23 +4,17 @@ const getOpeningHoursMessage = require('../lib/getOpeningTimesMessage');
 const midnightSpanCorrector = require('../lib/midnightSpanCorrector');
 const utils = require('../lib/utils');
 
-function sortByDistance(a, b) {
-  return a.dist - b.dist;
-}
-
 function filterServices(results, limits) {
   const openServices = [];
-  let serviceCount = 0;
-  let openServiceCount = 0;
-
-  const sortedServices = results.sort(sortByDistance);
   const now = getDateTime();
+  const servicesCount = results.length;
+  let openServiceCount = 0;
 
-  for (let i = 0; i < sortedServices.length; i++) {
-    const item = sortedServices[i];
+  for (let serviceCount = 0; serviceCount < servicesCount; serviceCount++) {
+    const item = results[serviceCount];
     const openingTimes = item.openingTimes;
-    let isOpen;
-    let openingTimesMessage;
+    let openingTimesMessage = 'Call for opening times';
+    let isOpen = false;
```

Comment:
The change is to provide defaults in order to remove the need for the existing else block. I think it simplifies the code and makes it clearer to see some stuff is being set without having to recognise it gets set to different values depending on the branch being traversed.